sdk/app_cpu1/common/sys/cmd/cmd_hw.c

```diff
@@ -17,6 +17,8 @@
 #include "sys/util.h"
 #include "usr/user_config.h"
 
+#include "xil_io.h"
+
```

Comment:
Curious why this change?

Reply:
removed, this is not necessary - I was directly reading from a register at one point but now everything is just a function call in `timing_manager.c`